src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx

```diff
@@ -82,15 +83,23 @@ const ActionDescription: FC<ActionDescriptionProps> = ({
           {loading ? (
             <div className="aspect-square h-[26px] w-auto rounded-full skeleton before:rounded-full" />
           ) : (
-            <UserAvatar
-              size={26}
-              userAddress={walletAddress}
-              userName={user?.profile?.displayName ?? undefined}
-              userAvatarSrc={
-                user?.profile?.thumbnail ?? user?.profile?.avatar ?? undefined
-              }
-              className="flex-shrink-0 flex-grow-0"
-            />
+            <UserInfoPopover
+              walletAddress={walletAddress}
```

Comment:
Maybe here we want to set also `withVerifiedBadge={false}`

Reply:
Ah good one! 👌